.editorconfig

```diff
@@ -0,0 +1,123 @@
+# To learn more about .editorconfig see https://aka.ms/editorconfigdocs
+###############################
+# Core EditorConfig Options   #
+###############################
+# All files
+[*]
+indent_style = space
+# Code files
+[*.{cs,csx,vb,vbx}]
+indent_size = 4
+insert_final_newline = true
+charset = utf-8-bom
```

Comment:
In other projects we are using `utf-8` for `[*]`. In some languages, the _BOM mark_ is a problem, in _C#_ it is not, but it is neither necessary.